src/libs/blueprint/conduit_blueprint_mpi_mesh.hpp

```diff
@@ -66,10 +66,22 @@ void CONDUIT_BLUEPRINT_API generate_index(const conduit::Node &mesh,
                                           Node &index_out,
                                           MPI_Comm comm);
 
+
+void CONDUIT_BLUEPRINT_API to_polygonal(const conduit::Node &n,
+                                   conduit::Node &dest,
+                                   const std::string& name);
+void CONDUIT_BLUEPRINT_API to_polyhedral(const conduit::Node &n,
+                                   conduit::Node &dest,
+                                   const std::string& name);
```

Comment:
On the one hand, I realize that these functions are peculiar because they apply mesh-wide transformations and thus don't fit nicely within an embedded namespace (e.g. `conduit::blueprint::mpi::mesh::topology::xyz::to_polygonal`), but I feel like more clarification is necessary to properly scope the set of functions to which this can be applied (i.e. structured AMR meshes). As an initial suggestion to start a conversation, would something like `from_amr_to_poly{gonal|hedral}` be an improvement?

Reply:
Could be `structured_to_polygonal`?  Strictly speaking, the input mesh doesn't have to be an AMR mesh.